packages/autorest.python/samples/specification/multiapi/generated/azure/multiapi/sample/v1/aio/operations/_multiapi_service_client_operations.py

```diff
@@ -230,14 +235,18 @@ async def begin_test_lro(
                 product=product,
                 content_type=content_type,
                 cls=lambda x, y, z: x,
+                stream=True,
                 headers=_headers,
                 params=_params,
                 **kwargs
             )
         kwargs.pop("error_map", None)
 
         def get_long_running_output(pipeline_response):
-            deserialized = self._deserialize("Product", pipeline_response)
+            _response = (
+                pipeline_response if getattr(pipeline_response, "context", {}) else pipeline_response.http_response
+            )
```

Comment:
see 1 and 2 of https://github.com/Azure/autorest.python/pull/2589#issue-2296719842.
-  I notice that pipeline_response with stream has no null `context` so here I use `getattr(pipeline_response, "context", {})` to judge with reference https://github.com/Azure/autorest.python/blob/1857677a4ddc47bf5f55b49e5546dbca027ed146/packages/autorest.python/samples/specification/multiapi/generated/azure/multiapi/sample/_serialization.py#L1559. Not sure whether there is better solution?
- for stream pipeline_response, I find that _deserialize [works](https://github.com/Azure/autorest.python/blob/1857677a4ddc47bf5f55b49e5546dbca027ed146/packages/autorest.python/samples/specification/multiapi/generated/azure/multiapi/sample/_serialization.py#L1566-L1567) with param `pipeine_response.http_response` both for sync and async API